types/highland/index.d.ts

```diff
@@ -1009,6 +1033,8 @@ declare namespace Highland {
 		 */
 		latest(): Stream<R>;
 	}
+
+	type MappingHint = number | string[] | Function;
```

Comment:
Is it possible to say `type MappingHint<T> = number | string[] | ((...args: any[]) => T)` (probably not, but one can dream...), because the return value of the function determines the type of the resulting stream. That is, strictly speaking, the types for mapping hint constructor is something like

`<R>(eventName: string, xs: NodeJS.EventEmitter, mappingHint?: MappingHint<R>): Highland.Stream<R>`

and the type for `wrapCallback` is

`wrapCallback<R>(f: (...args: any[]) => void, mappingHint?: MappingHint<R>: (...args: any[]) => Highland.Stream<R>`

where `R` is `any` if the mapping hint is either a `number` or a `string[]`.